Library/Homebrew/unpack_strategy/dmg.rb

```diff
@@ -87,8 +87,27 @@ def eject(verbose: false)
         return unless path.exist?
 
         if tries > 1
+          disk_info = system_command(
+            "diskutil",
+            args:         ["info", "-plist", path],
+            print_stderr: false,
+            verbose:      verbose,
+          )
+
+          # For HFS, just use <mount-path>
+          # For APFS, find the <physical-store> corresponding to <mount-path>
+          eject_path = if disk_info.success? && disk_info.plist.respond_to?(:fetch)
+            disk_info.plist
+                     .fetch("APFSPhysicalStores", [])
+                     .map { |store| store["APFSPhysicalStore"] }
+                     .compact
+                     .first || path
```

Comment:
```suggestion
                     .presence || [path]
```

Given `APFSPhysicalStores` is an array, let's loop through all paths.

Reply:
Added loop.
Not too sure if it will ever be more than 1 physical store, but shouldn't hurt.